services/web-server/README.md

```diff
@@ -120,8 +120,7 @@ UI_LOGIN_STRATEGIES='{
         "domain": "<auth0 subdomain>",
         "clientId": "<clientId from registration of client>",
         "clientSecret": "<clientSecret from registration of client>",
-        "audience": "api.sso.mozilla.com",
-        "scope": "display:all taskcluster-credentials openid profile"
+        "scope": "openid profile"
```

Comment:
There's no other acceptable value for this field, so I don't think it should be exposed as configuration.

Reply:
True that!

Reply:
still needs to be removed here..